cypress/support/commands.js

```diff
@@ -62,3 +62,71 @@ Cypress.Commands.add('loginUser', ({ email, password }) => {
     `utf8=%E2%9C%93&user%5Bemail%5D=${encodedEmail}&user%5Bpassword%5D=${encodedPassword}&user%5Bremember_me%5D=0&user%5Bremember_me%5D=1&commit=Continue`,
   );
 });
+
+const DEFAULT_AUTH_CONFIG = {
+  inviteOnlyMode: 0,
+  emailRegistration: 1,
+  allowedEmailDomains: 0,
+  publicEmailDomainList: '',
+  requireRecaptcha: 0,
+  recaptchaSiteKey: '',
+  recaptchaSecretKey: '',
+  authProvidersToEnable: '',
+  facebookKey: '',
+  facebookSecret: '',
+  githubKey: '',
+  githubSecret: '',
+  twitterKey: '',
+  twitterSecret: '',
+};
+
+/**
+ * Sets default values of SiteConfig atrributes relevant to Authentication Section.
+ *
+ * @param username {string} The username used in the test
+ * @param siteConfig
+ * @param siteConfig.inviteOnlyMode {boolean}
+ * @param siteConfig.emailRegistration {boolean}
+ * @param siteConfig.allowedEmailDomains {string}
+ * @param siteConfig.publicEmailDomainList {boolean}
+ * @param siteConfig.requireRecaptcha {boolean}
+ * @param siteConfig.recaptchaSiteKey {string}
+ * @param siteConfig.recaptchaSecretKey {string}
+ * @param siteConfig.authProvidersToEnable {string} Comma-separated string of providers to be enabled
+ * @param siteConfig.facebookKey {string}
+ * @param siteConfig.facebookSecret {string}
+ * @param siteConfig.githubKey {string}
+ * @param siteConfig.githubSecret {string}
+ * @param siteConfig.twitterKey {string}
+ * @param siteConfig.twitterSecret {string}
+ *
+ * @returns {Cypress.Chainable<Cypress.Response>} A cypress request for setting SiteConfig values for the Authentication Section.
+ */
+Cypress.Commands.add(
```

Comment:
This function was thus generalized to allow test-writers pre-configure Authentication Settings that make sense for their tests.